Update release.rst for #137 (#138)
ACCESS-NRI/access-nri-intake-catalog: docs/management/release.rst

following all steps below. Steps 1 and 2 below should be done in a PR and merged before commencing step 3. If the release does 
not include an update to the catalog on Gadi, skip the first two steps below:

#. [IF UPDATING THE CATALOG] Create a new version of the catalog on Gadi (this will take about 1 hour)
   ::
      $ export RELEASE=vX.X.X
      $ cd bin
      $ qsub -v version=${RELEASE} build_all.sh
   .. note:: 

>       If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the `access_nri_intake.source` and `access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of `access_nri_intake`

Reviewer: ```suggestion
      If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the :code:`access_nri_intake.source` and :code:`access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of :code:`access_nri_intake`
```

Author: Oh I just re-read this. Is the 'using the release version of access_nri_intake' correct? In your comment above I thought it need to be the development version ?

Reviewer: Release version is right

      $ export RELEASE=vX.X.X
      $ cd bin
      $ qsub -v version=${RELEASE} build_all.sh

>    #. If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To workaround::

Reviewer: ```suggestion
   #. If the `schema <https://github.com/ACCESS-NRI/schema>`_ has changed, or you have not used the intake catalog recently, this step may fail with a *Network is unreachable* error trying to download the schema json files. To download and cache the schema, first import the `access_nri_intake.source` and `access_nri_intake.catalog` sub-packages from a Gadi node with network access (e.g. a login or ARE node). I.e., using the release version of `access_nri_intake`::
```

Reviewer: Also, this could go in a `.. note::` block - see [here](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#notes)

>       $ module use /g/data/hh5/public/modules/; module load conda/analysis3

Reviewer: ```suggestion
```

Reviewer: This will not download the correct version of the schema if it has been changed. Users should load their development environment
